src/components/resource-selector.js

```diff
@@ -42,7 +42,7 @@ function ResourceSelector({ allowedLocationTypes, initialUrl, onCancel, onSelect
             initialUrl={initialUrl}
             onEnterFullscreen={() => setIsFullscreen(true)}
             onExitFullscreen={() => setIsFullscreen(false)}
-            onSelect={onSelect}
+            onSelect={filePath => onSelect(filePath)}
```

Comment:
I don't understand the change here, it's equivalent :thinking:

Reply:
I know, just wanted to make the argument transparent, cause I was not sure if this will be the value we stick with.
Should I remove it?

Reply:
Hm, we usually just propagate regardless of the arguments. I personally would remove it, but I'm not at all passionate about it. Up to you, I was just curious.

Reply:
Ok, removed! It's true that if we'd change the method signature we'd have to change also here (which always leaves room for bugs).